### DISCUSSION TITLE
DefaultDate Option in DateTimePicker

### DISCUSSION THREAD

**Discussion by sssaha** [snehilvj/dash-mantine-components#449] [Q&A]
The Mantine DatePicker allows to set a default Date Time value so the calendar does not show up from current date. How that parameter can be passed to the DateTimePicker object? Is that possible

**Comment by AnnMarieW** [comment]
Use the `value` prop.  It takes either a date time string or object

```


component = dmc.DateTimePicker(
    clearable=True,
    value="2023-12-25 10:30:00",
    label="Pick date and time (clearable)",
    placeholder="Pick Date and time",
)

```

**Comment by AnnMarieW** [comment]
Hi @sssaha 

Since this is more of a "how to" question, I'm converting it to a discussion.  If it turns out to be a feature request or a bug, we can open an issue for it later.
